gems/aws-sdk-core/lib/seahorse/client/net_http/connection_pool.rb

```diff
@@ -339,8 +340,8 @@ def __setobj__(obj)
 
           # Sends the request and tracks that this session has been used.
           def request(*args, &block)
-            @last_used = Time.now
             @http.request(*args, &block)
+            @last_used = Aws::Util.monotonic_milliseconds
```

Comment:
`last_used` is changing types from Time to an integer, be aware that anyone depending on `last_used` (see line 330 attr_reader) will be broken. At a minimum, the documentation should be updated, but is there a way to avoid that breakage?

Reply:
Good point! I didn't think about this as something exposed and used outside of the gem scope.

I can think of a few options:
- 
- Updating the [doc](https://github.com/aws/aws-sdk-ruby/blob/28b97fce0df2d5d597077834d780a9ee149d9727/gems/aws-sdk-core/lib/seahorse/client/net_http/connection_pool.rb#L330) to complete the change.
- Keeping track of and exposing both clocks - wall and monotonic. 
- Removing this change altogether. It's a marginal improvement compared to the risk of breaking someone's code.

Reply:
This class is marked as @api private so it should be fair game to change -- could you update the documentation for it at least?

Reply:
The attribute type is now reflected in the doc.